CONTRIBUTING.md

```diff
@@ -101,3 +101,10 @@ rule is also loosened, too.
 Exceptionally, commit type `docs` may omit scope. e.g, docs: update contracts/README.md.
 If the change affects more than one scope, the commit scope may be omitted.
 e.g, feat: add Colosseum contract
+
+## How to review
+
+If a `Request Change` is raised during a PR review, there's nothing to be addressed.
+However, if you receive a comment, it is advisable for the comment author to address it.
+Once the PR author has incorporated all the requested changes, please mention
+the comment author and inform them so that they can review and `Approve` the changes.
```

Comment:
```suggestion
If a PR author receives `Comment` from reviewers, not `Request changes`, it is recommended to wait for the comment
authors to confirm the changes that the PR author made after the review. Therefore, once the PR author has reflected all
the requested changes, please re-request for the comment authors to review again so that they can review and `Approve`
the changes. The PR author should merge and close the PR after receiving `Approve` from all comment authors.
```

Reply:
> If a `Request Change` is raised during a PR review, there's nothing to be addressed.

This sentence looks weird to me because `Request Changes` imply that the PR **should** address something before merge. The description @seolaoh suggested looks better to me.

Reply:
@therocks How do you think?

Reply:
In general, Sara(@seolaoh)'s suggestion  looks more clear to me. 
But,I have a minor confusion regarding 'Request Change'. Do you use 'Request Change' for meaning that there are some changes in pull request? If you are using the word for meaning that there is a request for changes, than 'Change Request' or 'Request for Changes' more clear to me.

Reply:
@joonkeep Could you help us about what @therocks comments above?

Reply:
Since the buttons name is explicitly "Comment" and "Request changes" when doing PR review, I put the terms in quotes so that others would implicitly know what the terms meant.

Reply:
@seolaoh @0xHansLee @chokobole @therocks 

There are only three options a reviewer can choose: Comment, Approve, and Request Changes. I think we should use each option with its underlying meaning, not any other way around. In my view, @chokobole opened this PR to make sure a PR author do not merge a PR even when there are unresolved comments, not to make a new rule for 'Request Changes' option.

Also, I think this discussion is about 'how to merge', not 'how to review'.

To sum up, I suggest our rules for 'how to merge' as below:

===

## Recommendations for Merging a PR with Unaddressed Comments

While a "Comment" does not necessarily prevent the PR author from merging the PR, it is advisable to fully address any discussions or decisions related to the comment before proceeding. The PR author should ensure that all decisions made in the comments have been implemented. After incorporating these changes, the author should request a review from all the commenters involved, so they can review the updated version and provide their approval.

The PR author should proceed with merging the PR only after receiving approval from all the commenters.

Reply:
FYI, This is joonki's recommendation.

```
If a PR author receives a `Comment` from reviewers, not a `Request Changes`, the PR Author should 
wait for the comment authors to confirm the changes that the PR author made after the review. 
Once the PR author has reflected all of the requested changes, please make another request 
to the comment authors to review and `Approve` the changes. The PR author should merge and
close the PR after receiving `Approve` from all comment authors.
```

Reply:
Joonkee's version still has the term 'Request Changes' in the sentences, which I do not think necessary. Do we really need to include this redundant term?